app/views/users/two_factor_setup/authenticator_setup/new.html.haml

```diff
@@ -0,0 +1,40 @@
+%article.verify
+  %h1 Google Authenticator setup
+
+  %h2 Step 1
+  %p
+    Enter this code into your authenticator app
+
+  .callout
+    %h3= @totp_secret
+
+  %section.sign-in-form
+    .sign-in-footer-links
+      .background-border-heading
+        %span.inner
+          Or
+
+  .qrcode
```

Comment:
I think the qrcode should be displayed first. Would anyone ever manually type the code in??